wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Field.kt

```diff
@@ -22,7 +22,12 @@ import com.squareup.wire.schema.internal.parser.OptionElement.Companion.PACKED_O
 import kotlin.jvm.JvmStatic
 
 data class Field(
-  val packageName: String?,
+  // The namespace in which the field is defined. For top-level extensions,
+  // the namespace is the package of the file in which the extension is
+  // declared or null if the file has no package. For other fields and
+  // extensions defined inside a message, this is the fully-qualified name
+  // of the enclosing message.
```

Comment:
Could we remove `The namespace in which the field is defined.` and make the comment a documentation one with `**` ?

Reply:
Doh, my bad. I've been writing too much Go lately (where doc comments use `//`). Will fix.